modules/webbrowser/src/properties/propertycefsynchronizer.cpp

```diff
@@ -80,8 +85,6 @@ void PropertyCefSynchronizer::OnLoadError(CefRefPtr<CefBrowser> browser, CefRefP
         ss << "<br/>Description: " << errorText.ToString();
     }
 
-    ss << "</body></html>";
```

Comment:
any particular reason why the closing tags are removed?

Reply:
Hmm, looks like a mistake. Should be reverted. Good catch